src/components/main.vue

```diff
@@ -44,7 +41,7 @@
       const showRecommendations: ComputedRef<boolean> = computed(
         () => $x.noResults && !$x.partialResults.length && !$x.semanticQueries.length
       );
-      return { hasSearched, semanticsLocation, showRecommendations };
+      return { hasSearched, semanticsLocation, showRecommendations, x: use$x() };
```

Comment:
```suggestion
      return { hasSearched, semanticsLocation, showRecommendations, x: $x };
```

Although I have mixed feelings.